docker-compose.yml.img

```diff
@@ -35,26 +37,30 @@ services:
       WEB_URL: '${MONOCLE_URL:-http://localhost:3000}'
     command: uwsgi --uid guest --gid nogroup --http :9876 --socket :9877 --manage-script-name --mount /app=monocle.webapp:app
     volumes:
-      - $PWD/etc:/etc/monocle:z
+      - ./etc:/etc/monocle:z
     healthcheck:
         test: python -c "import requests,sys; r=requests.get('http://localhost:9876/api/0/health'); print(r.text); sys.exit(1) if r.status_code!=200 else sys.exit(0)"
 
+  # ELASTIC_CONTAINER_URL is defined in .env file.
   elastic:
-    image: "docker.elastic.co/elasticsearch/elasticsearch:7.10.1"
+    image: ${ELASTIC_CONTAINER_URL}
```

Comment:
I think we need to keep the default to elasticsearch image if no .env file provided.